saleor/graphql/giftcard/mutations.py

```diff
@@ -99,7 +100,7 @@ def clean_input(cls, info, instance, data):
         cls.clean_expiry_settings(cleaned_input, instance)
         cls.clean_balance(cleaned_input, instance)
 
-        cleaned_input.get("user_email", None)
+        # TODO: validate user email
```

Comment:
It will be done in separate PR?

Reply:
Yep, email should be validated in couple of these mutations, will be added later.